node/src/accountant/mod.rs

```diff
@@ -1616,11 +1620,12 @@ mod tests {
             agent: Box::new(agent),
             response_skeleton_opt: Some(response_skeleton),
         };
-        let analyzed_accounts = convert_collection(unadjusted_qualified_accounts.clone());
+        let analyzed_accounts =
+            convert_qualified_into_analyzed_payables_in_test(unadjusted_qualified_accounts.clone());
```

Comment:
Let's define this function in test utils, and there should be a comment in this function mentioning that this conversion shouldn't be done in the production code.

One that's done, there won't be a need to use the suffix _in_test anymore.